src/components/Comment/Comment.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { TypeOfComment } from '../../types';
+import './Comment.scss';
+
+export const Comment = ({ name, body, email }) => (
+  <div className="comment">
+    <p>
+      <a href={`mailto:${email}`}>
+        {email}
+      </a>
+    </p>
+    <p className="comment__name">{name}</p>
+    <p>{body}</p>
+  </div>
+);
+
+Comment.propTypes = TypeOfComment.isRequired;
```

Comment:
```suggestion
Comment.propTypes = {
  name: PropTypes.string.isRequired,
  body: PropTypes.string.isRequired,
  email: PropTypes.string.isRequired,
};
```
If you've destructured comment object and pass separate props, you need to describe their types separately.